pkg/autoscaler/aggregation/aggregation.go

```diff
@@ -19,12 +19,12 @@ package aggregation
 import "time"
 
 // Accumulator is a function accumulating buckets and their time.
-type Accumulator func(time time.Time, bucket float64Bucket)
+type Accumulator func(time time.Time, bucket *float64Value)
```

Comment:
is `bucket` still an accurate name for the second parameter?

Reply:
yeah.